libs/atqr/domain/src/lib/valueObjects/notification.ts

```diff
@@ -0,0 +1,114 @@
+import { ValueObject } from '../../utils/valueObject';
+import { Guid } from '@tokilabs/lang';
+import { NotificationLogEntry } from './notificationLogEntry.valueObject';
+import { MessageTemplateId } from '../services';
+
+enum NotificationCategory {
+  EuDuvidoProductUpdates = 'EuDuvidoProductUpdates',
+  EuDuvidoPromotions = 'EuDuvidoPromotions',
+  NewInvitations = 'NewInvitations',
+  ChallengeUpdates = 'ChallengeUpdates',
+}
+
+export enum NotificationChannel {
+  Email = 'Email',
+  SMS = 'SMS',
+  Push = 'Push',
+}
+
+export class Notification extends ValueObject<Notification> {
+  constructor(
+    public readonly id: Guid,
+    public readonly category: NotificationCategory,
+    public readonly templateId: MessageTemplateId,
+    public readonly allowedChannels: NotificationChannel[],
+    public readonly createdAt: Date,
```

Comment:
Disl says this should be automatically set to "Today". You don't need to have the parameters in the same order as defined in the Disl definition.